contracts/PolygonZkEVM.sol

```diff
@@ -425,9 +441,11 @@ contract PolygonZkEVM is
     /**
      * @notice Allows a sequencer to send multiple batches
      * @param batches Struct array which holds the necessary data to append new batches to the sequence
+     * @param feeRecipient Address that will receive the fees from L2
```

Comment:
So I'm guessing when L2 users pay tx fees, the sequencer can opt to be paid out to any arbitrary fee recipient, similar to how L1 works when proposing blocks?

Reply:
Yes exactly! this becomes even more useful if the trustedSequencer address gets managed by a SC, With some kind of consensus, imagine there are slots of time. Then any sequencer will be able to call sequenceBAtches through the smart contract and choose their fee recipeint